addons/point_of_sale/models/pos_note.py

```diff
@@ -10,6 +10,7 @@ class PosNote(models.Model):
 
     name = fields.Char(required=True)
     sequence = fields.Integer('Sequence', default=1)
+    color = fields.Integer(string='Color')
```

Comment:
maybe now would be the time to change the orderline model to have a many2one with `pos.note`

Reply:
@vlst-odoo  can we create a new task for this point?